go/adbc/driver/bigquery/driver.go

```diff
@@ -77,6 +77,49 @@ const (
 
 	AccessTokenEndpoint   = "https://accounts.google.com/o/oauth2/token"
 	AccessTokenServerName = "google.com"
+
+	// UseAppDefaultCredentials instructs the driver to authenticate using
+	// Application Default Credentials (ADC).
+	OptionValueAuthTypeAppDefaultCredentials = "app_default_credentials"
+)
+
+// option that can be passed to the NewClient method.
+const (
+	// WithAccessToken instructs the driver to authenticate using the given
+	// access token.
+	WithAccessToken = "adbc.google.bigquery.auth.access_token"
```

Comment:
Where is this actually used in the driver?

Reply:
I want to address the auth type in another PR.